src/incident.coffee

```diff
@@ -13,8 +13,9 @@
 #   HUBOT_PAGERDUTY_SUBDOMAIN
 #   
 # Commands:
-#   hubot incident start - Initiate incident
-#   hubot incident end - Resolve incident
+#   hubot incident start <pagerduty id> - Manually initiate incident
```

Comment:
Why do we pass a pagerduty id? This seems a little redundant, are we creating incidents with specific IDs?